win32/GNUmakefile

```diff
@@ -487,6 +487,15 @@ GCCVER2		:= $(shell for /f "delims=. tokens=1,2,3" %%i in ('$(GCCBIN) -dumpversi
 GCCVER3		:= $(shell for /f "delims=. tokens=1,2,3" %%i in ('$(GCCBIN) -dumpversion') do echo %%k)
 endif
 
+# are we using UCRT or MSVCRT?
+ifeq ($(CCTYPE),GCC)
+CRT_TYPE := $(shell gcc --version 2>&1 | find "msvcrt" >nul && ( echo msvcrt) || ( echo ucrt) )
+else ifeq ($(CCTYPE),MSVC120)
+CRT_TYPE := msvcrt
+else
+CRT_TYPE := ucrt
+endif
+
```

Comment:
You can check for `_UCRT` define instead. Both Visual C++ and mingw define it.

Reply:
Thanks, my google-fu failed me when I tried finding such a macro.